packages/react-admin-ui/src/organisms/Footer.tsx

```diff
@@ -1,10 +1,70 @@
 import clsx from 'clsx';
-import { WithClassName, WithContent } from '../withs';
+import { useTranslation } from 'react-i18next';
+import Row from '../atoms/Row';
+import Text from '../atoms/Text';
+import { rich_text } from '../types';
+import { Image } from '../atoms/Image';
+import { WithBox, WithDescription, WithLogo } from '../withs';
+import { Block, Icon } from '../atoms';
+import Link from '@material-ui/core/Link';
 
-export function Footer({ className, content }: FooterProps) {
-    return <div className={clsx('p-4 text-center text-sm', className)}>{content}</div>;
+export function Footer({
+    logo,
+    color = 'dark',
+    variant = 'contained',
+    contact,
+    links,
+    description,
+    copyright,
+}: FooterProps) {
+    const { t } = useTranslation();
+    return (
+        <Block color={color} variant={variant} corner={'square'}>
+            <Row className={clsx('p-8 xs:flex-col')}>
+                <div className={'flex-1 flex flex-col'}>
+                    {logo && <Image {...logo} className={'m-6 w-24 h-24 ml-1 rounded-full'} expand={false} />}
+                    <div className={'mt-3 justify-end'}>
+                        <Text className={'mt-2 leading-loose'} variant={'overline'} text={description} />
+                    </div>
+                    <Text className={'mt-2'} variant={'overline'} text={copyright} />
+                </div>
+                <div className={'flex mt-6 sm:flex-col'}>
+                    <div className={'px-8 xs:pt-8'}>
+                        <Text className={'mb-6'} variant={'section'} text={t('project_name')} />
+                        {links.map((link) => (
+                            <Link color="inherit" href={link.target} className={'block leading-loose'}>
```

Comment:
il manque le `key = `, car en effet, a chaque fois que l'on utilise `.map(` pour créer en boucle des composants React, on doit spécifier une valeur pour key
https://fr.reactjs.org/docs/lists-and-keys.html#keys